internal/cmd/agent/deployer/deployer.go

```diff
@@ -177,14 +177,14 @@ func (d *Deployer) fetchNamespace(ctx context.Context, releaseID string) (*corev
 	return &list.Items[0], nil
 }
 
-// addLabelsFromOptions updates nsLabels so that it only contains all labels specified in optLabels, plus the `name` labels added by Helm when creating the namespace.
+// addLabelsFromOptions updates nsLabels so that it only contains all labels specified in optLabels, plus the `kubernetes.io/metadata.name` labels added by Helm when creating the namespace.
```

Comment:
There's a discrepancy between this comment and the one on L187: is the label added by Helm or Kubernetes? :þ